features/environment.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Git Town performs correctly depending on the environment
+
+  Scenario Outline: Git Town commands run outside of a Git repository
+    Given I'm currently not in a git repository
+    When I run `<COMMAND>` while allowing errors
+    Then I see "error: This is not a git repository."
+    And I don't see "fatal: Not a git repository"
```

Comment:
Since `Then I see` is now checking for exact output, there is no need to test that you don't see anything else